core/db/entities.py

```diff
@@ -41,7 +41,8 @@ class Submission(BaseModel):
 
     __tablename__ = "submission_dim"
 
-    submission_id = sqla.Column(sqla.String(), nullable=False, unique=True)
+    # TODO: Change submission ID from an int value to a string that includes programme or fund type?
+    submission_id = sqla.Column(sqla.Integer(), nullable=False, unique=True)
```

Comment:
see previous comment. We should have enough info to build this. Doesn't need to reference programme or project, since they will actually reference this "parent" submission ID. Fund type can always be looked up via db queries to Programe_Ref.fund_type, don't need it in the submission id